homeassistant/components/cloud/__init__.py

```diff
@@ -65,7 +76,7 @@
         vol.Optional(CONF_REGION): str,
         vol.Optional(CONF_RELAYER): str,
         vol.Optional(CONF_ALEXA): ALEXA_SCHEMA,
-        vol.Optional(CONF_GOOGLE_ACTIONS): ASSISTANT_SCHEMA,
+        vol.Optional(CONF_GOOGLE_ACTIONS): GACTIONS_SCHEMA,
```

Comment:
I noticed this over on the Alexa PR, the rename from Assistant to Actions. What's the motivation there? I ask because I'd originally written this as actions and went with assistant because it felt like that matches what end users will expect the component to be named (only devs see it as Actions on Google).

(Also if I wanted to nitpick I'd say all the assistant->actions renames should be in this PR)

Reply:
That's only within the Cloud component. I did not want to change the component name. I called it that way because that seems to be the product name that Google gave their side.